src/navigation/breadcrumbs/__test__/breadcrumbsMockups.js

```diff
@@ -0,0 +1,45 @@
+export default { // Router mockup
```

Comment:
Used for unit testing and some breadcrumbs docs, I don't think we should reference files outside the docs folder in the DOCS code snippets but I already asked about that previously in [here](https://github.com/saddlebackdev/react-cm-ui/pull/195/files#r477450845), please let me know, thanks.

Reply:
It seems like we've resolved this, right?  With the simplified routes used in the examples?